src/pages/dashboard/user/components/userInfo.js

```diff
@@ -34,18 +35,15 @@ function UserInfo({
       <Row>
         <Col span={20}>
           <Row className="info-avatar">
-            <Avatar
-              size={75}
-              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
-            />
+            <Avatar size={75} src={profilePicture} alt="Profile Picture" />
           </Row>
           <Row className="info-detail">
-            {data.map((agent) => (
-              <Col span={24} key={agent.field}>
+            {data.map((user) => (
+              <Col span={24} key={user.field}>
```

Comment:
next pakein `user?.field` atau default value biar ga nge break kalau misalkan undefined, dan kalau misalkan key nya banyak, langsung di deconstruct aja
```suggestion
            {data.map(({field = ''}) => (
              <Col span={24} key={field}>
```

Reply:
Baik Mba